### ISSUE TITLE
Mixed css and scss sourcemaps in inspector

### ISSUE THREAD

**Issue by ivancuric** [dlmanning/gulp-sass#276]
Here's the build I'm using:

``` javascript
gulp.task('sass', function() {
  return gulp.src(files.scss)
    .pipe(plumber())
    .pipe(sourcemaps.init())
    .pipe(sass())
    .on('error', notify.onError({
      message: function(error) {
        return error.message;
      }
    }))
    .pipe(combineMq())
    .pipe(minifyCSS({
      keepSpecialComments: 0,
      keepBreaks: false,
      roundingPrecision: 5
    }))
    .pipe(autoprefixer())
    .pipe(sourcemaps.write())
    .pipe(gulp.dest('farmacia/static/css/'))
    .pipe(filter(files.css))
    .pipe(browserSync.reload({
      stream: true
    }));
});
```

And here's what Chrome sees in the inspector:

![screen shot 2015-05-14 at 13 46 28](https://cloud.githubusercontent.com/assets/2827047/7631280/10937e1c-fa40-11e4-801b-8a678f58bacc.png)

The workspace is added correctly, files are mapped, however only some rules get mapped to the corresponding .scss.

**Comment by Snugug** [comment]
Please ensure you've updated to the latest version (there's a fix for working with with sourcemaps and autoprefixer in there) and reduce your gulp task to the bare minimum that we can reproduce this problem, along with the bare minimum Sass to reproduce this problem

**Comment by Snugug** [comment]
Will close if we haven't heard back

**Comment by TrySound** [comment]
@all #314

**Comment by Snugug** [comment]
Resolved by #314
